generators/entity/index.js

```diff
@@ -83,13 +82,27 @@ module.exports = EntityGenerator.extend({
             defaults: ''
         });
 
+        // This adds support for a `--skip-server` flag
+        this.option('skip-server', {
+            desc: 'Skip the server-side application generation',
+            type: Boolean,
+            defaults: false
+        });
+
         this.regenerate = this.options['regenerate'];
         this.entityTableName = this.options['table-name'] || this.name;
+        this.entityName = _s.capitalize(this.name);
         this.entityTableName = _s.underscored(this.entityTableName).toLowerCase();
         this.entityAngularJSSuffix = this.options['angular-suffix'];
-        if (this.entityAngularJSSuffix && !this.entityAngularJSSuffix.startsWith('-')) {
+        if(!_.isUndefined(this.config.get('skipServer'))) {
```

Comment:
You're not I need to learn this shorten syntax, I was used to use `if/else` at school.